### ISSUE TITLE
Consensus node stop block trigger 0 fast rate not punished

### ISSUE THREAD

**Issue by qiusese** [PlatONnetwork/PlatON-Go#699]
PlatON version: `0.7.4 version`
OS & Version: Windows/Linux/OSX

1、Initialize 4 consensus nodes
2、Stop one of the node processes
3、Waiting for two consensus rounds
4、The node triggers 0 fast rate penalty, but the penalty amount is not deducted

![image](https://user-images.githubusercontent.com/48665478/68365349-5e184a00-016b-11ea-8928-25eb363de4c5.png)

**Comment by GavinXu520** [comment]
This bug was caused by a code change before, the reason is when calculating the penalty amount, an internal variable with the same name is used, which leads to the `variable masking'. Specifically:
```go
slashAmount := common.Big0

				canMutable, err := stk.GetCanMutableByIrr(validator.NodeAddress)
				if nil != err {
					log.Error("Failed to BeginBlock, call candidate mutable info is failed", "blockNumber", header.Number.Uint64(), "blockHash", blockHash.TerminalString(), "err", err)
					return err
				}
				totalBalance := calcCanTotalBalance(header.Number.Uint64(), canMutable)
				if blockReward > 0 {
                                        // fix here
					slashAmount := calcSlashBlockRewards(header.Number.Uint64(), uint64(blockReward), state)
					if slashAmount.Cmp(totalBalance) > 0 {
						slashAmount = totalBalance
					}
				}

```

fix to 
```go
slashAmount := common.Big0

				canMutable, err := stk.GetCanMutableByIrr(validator.NodeAddress)
				if nil != err {
					log.Error("Failed to BeginBlock, call candidate mutable info is failed", "blockNumber", header.Number.Uint64(), "blockHash", blockHash.TerminalString(), "err", err)
					return err
				}
				totalBalance := calcCanTotalBalance(header.Number.Uint64(), canMutable)
				if blockReward > 0 {
                                          // fix here
					slashAmount = calcSlashBlockRewards(header.Number.Uint64(), uint64(blockReward), state)
					if slashAmount.Cmp(totalBalance) > 0 {
						slashAmount = totalBalance
					}
				}
```

**Comment by benbaley** [comment]
This issue had fixed in #700
